scripts/release/asset_publish.py

```diff
@@ -277,33 +286,66 @@ def _str2bool(v: str) -> bool:
             )
 
             # Handle specific asset types
-            if asset.type in [assets.AssetType.COMPONENT, assets.AssetType.ENVIRONMENT]:
-                # Assemble command
-                cmd = assemble_command(
-                    asset.type.value, str(asset.spec_with_path),
-                    registry_name, final_version, resource_group, workspace, debug_mode)
-                # Run command
-                run_command(cmd, failure_list, debug_mode)
+            if asset.type == assets.AssetType.COMPONENT:
+                # load component and check if environment exists
+                logger.print(f"spec's path: {asset.spec_with_path}")
+                component = load_component(asset.spec_with_path)
+                env = component.environment
+                ws_env_pattern = re.compile("^(.+):(.+)$")
```

Comment:
Consider adding support for `azureml:<env_name>@latest`?

Reply:
To fetch env or model in registry using latest tag is still not supported. I get following error when tried: 

![image](https://user-images.githubusercontent.com/61145377/218706675-1c5f4e71-9f25-4b85-808e-302538c5b2dc.png)

Reply:
This could be in a later PR, though. I'm just saying that this feature could be fleshed out even more by providing support for `@latest`.

Reply:
Yeah make sense. I can add support label in next PR

Reply:
Is this taken care of or are we targeting in the next PR?

Reply:
Updated the regex for both ws and registry to consider labels as well. But to be able to fetch env asset using latest label we will need to wait for support in SDK. Added a bug here for tacking: #415